app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  skip_before_action :verify_authenticity_token
```

Comment:
Ok, I have some question here.
1. What happen if in the future we add some feature out of the API? As a backoffice for example? 
2. You did not any trouble with `protect_from_foregery` ?

Reply:
1. If in a future we need to add some backoffice features, we can create two parents controllers: One for the API controllers and the other one for the Backoffice controllers

2. I removed the skip before action and added protect_from_forgery with: :null_session

Reply:
Ok perfect, remenmber point 1 for the future. Well done!!